src/xinetd.xml

```diff
@@ -0,0 +1,58 @@
+<SPDX name="xinetd License" identifier="xinetd" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/Xinetd_License</url>
+  </urls>
+  <license>
+    <optional>
+      <p>ORIGINAL LICENSE: This software is</p>
+    </optional>
+    <copyright>
+      <p>(c) Copyright 1992 by Panagiotis Tsirigotis</p>
+    </copyright>
+    <body>
+      <p>The author (Panagiotis Tsirigotis) grants permission to use, copy, and distribute this software and its
+         documentation for any purpose and without fee, provided that the above copyright notice extant in
+         files in this distribution is not removed from files included in any redistribution and that this
+         copyright notice is also included in any redistribution.</p>
+      <p>Modifications to this software may be distributed, either by distributing the modified software or by
+         distributing patches to the original software, under the following additional terms:</p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>The version number will be modified as follows:</p>
+        </li>
+      </list>
+      <list>
+        <li>
+          <b>a.</b>
+          <p>The first 3 components of the version number (i.e &lt;number&gt;.&lt;number&gt;.&lt;number&gt;)
+             will remain unchanged.</p>
+        </li>
+        <li>
+          <b>b.</b>
+          <p>A new component will be appended to the version number to indicate the modification level. The
+             form of this component is up to the author of the modifications.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>The author of the modifications will include his/her name by appending it along with the new
+             version number to this file and will be responsible for any wrong behavior of the modified
+             software.</p>
+        </li>
+      </list>
+      <p>The author makes no representations about the suitability of this software for any purpose. It is
+         provided "as is" without any express or implied warranty.</p>
+    </body>
+    <optional>
```

Comment:
this optional block looks like a note rather than an optional piece of text. but it's included at http://spdx.org/licenses/xinetd.html and at https://fedoraproject.org/wiki/Licensing/Xinetd_License 

should we remove, or at least move to notes?

it seems like matching should still work without including this, even if users included this extraneous text in their source code, no?